llvm/include/llvm/ADT/DenseMap.h

```diff
@@ -1106,10 +1112,14 @@ class SmallDenseMap
     new (getLargeRep()) LargeRep{nullptr, 0};
   }
 
-  void grow(unsigned AtLeast) {
-    if (AtLeast > InlineBuckets)
-      AtLeast = std::max<unsigned>(64, NextPowerOf2(AtLeast - 1));
+  static unsigned computeNumBuckets(unsigned NumBuckets) {
```

Comment:
The name is confusing a bit. (What does it compute if NumBuckets is already the input to the function?)

Reply:
Maybe rename the parameter to MinNumBuckets (or to AtLeast, the original name, not sure which one conveys the intent clearer)?